go_api/domain/admin.go

```diff
@@ -0,0 +1,15 @@
+package domain
+
+import (
+	"time"
+)
+
+type Admin struct {
+	ID        int       `json:"id" gorm:"primary_key;not null"`
+	Name      string    `json:"name" gorm:"not null"`
+	Email     string    `json:"email" gorm:"not null"`
+	Password  string    `json:"password" gorm:"not null"`
+	CreatedAT time.Time `json:"created_at" gorm:"not null"`
+	UpdatedAT time.Time `json:"updated_at" gorm:"not null"`
+
+}
```

Comment:
```suggestion
}

```
最後の行は空行入れましょう！いくつか見られるので修正お願いします
自動で最後に入れる機能がVscodeにはあるので設定していなかったら設定を推奨します！
https://memo.koumei2.com/vscode-%E3%81%AE%E6%94%B9%E8%A1%8C%E9%96%A2%E9%80%A3%E3%81%AE%E5%9F%BA%E6%9C%AC%E8%A8%AD%E5%AE%9A/